CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 1.0.2 (Next)
```

Comment:
Since we're adding a major feature, increment the version to `1.1.0` here and in `version.rb`.

Reply:
I did.